npc/re/mobs/dungeons/odin.txt

```diff
@@ -54,3 +54,18 @@ odin_tem03,0,0	monster	Valkyrie	1765,2,5400000,1800000
 odin_tem03,0,0,0,0	boss_monster	Valkyrie Randgris	1751,1,28800000,600000,1
 odin_tem03,0,0	monster	Blue Plant	1079,18,1800000,900000
 odin_tem03,0,0	monster	Shining Plant	1083,18,1800000,900000
+
+
+//==================================================
+// odin_past - Ancient Odin Shrine
+//==================================================
+odin_past,0,0	monster	Angelgolt	20604,60,5000
+odin_past,0,0	monster	Angelgolt	20605,60,5000
+odin_past,0,0	monster	Holy Frus	20606, 45,5000
+odin_past,0,0	monster	Holy Skogul	20607, 45,5000
+odin_past,0,0	monster	Arch Plasma	20608,5,5000
+odin_past,0,0	monster	Spectral Plasma	20609,5,5000
+odin_past,0,0,0,0	boss_monster	Valkyrie Reginleif	20610,1,28800000,600000,1
+odin_past,0,0,0,0	boss_monster	Valkyrie Ingrid	20611,1,28800000,600000,1
+odin_past,0,0	monster	Blue Plant	1079,18,1800000,900000
+odin_past,0,0	monster	Shining Plant	1083,18,1800000,900000
```

Comment:
The spawn number is different from https://www.divine-pride.net/database/map/odin_past
Do you mind to share your source?

Reply:
Hmmm, well, I didn't realized which spawn was wrong. I just copied the spawn from Odin Temp 03. I'll fix it according to DP

Reply:
Not sure where is correct info, but korean navi luas contain next data:
There are few differences in amount of mobs.
(respawn time is dummy)
> odin_past,0,0	monster	SHINING_PLANT	1083,2,5000,0
odin_past,0,0	monster	BLUE_PLANT	1079,5,5000,0
odin_past,0,0	monster	ANGELGOLT	20604,40,5000,0
odin_past,0,0	monster	ANGELGOLT2	20605,60,5000,0
odin_past,0,0	monster	PLASMA_SPT	20609,5,5000,0
odin_past,0,0	monster	PLASMA_ARCH	20608,5,5000,0
odin_past,0,0	monster	HOLY_FRUS	20606,45,5000,0
odin_past,0,0	monster	HOLY_SKOGUL	20607,45,5000,0
odin_past,0,0	boss_monster	REGINLEIF	20610,1,10800000,600000,0
odin_past,0,0	boss_monster	INGRID	20611,1,10800000,600000,0

Reply:
Seems to be correct. I'll fix it when I get home.